operator/internal/decommissioning/statefulset_decomissioner.go

```diff
@@ -324,73 +327,74 @@ func (s *StatefulSetDecomissioner) Decommission(ctx context.Context, set *appsv1
 		return false, nil
 	}
 
-	unboundVolumeClaims, err := s.findUnboundVolumeClaims(ctx, set)
-	if err != nil {
-		log.Error(err, "error finding unbound PersistentVolumeClaims")
-		return false, err
-	}
+	setCacheKey := client.ObjectKeyFromObject(set)
+	if s.cleanupPVCs {
```

Comment:
@andrewstucki i added this condition here.
some code references the cache even if PVC cleanup is disabled, but i think it's not an issue.